tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -1726,6 +1727,23 @@ trait WebActionsApiBaseTests extends ControllerTestCommon with BeforeAndAfterEac
         status should be(BadRequest)
       }
     }
+
+    it should s"support json (including +json subtypes) (auth? ${creds.isDefined})" in {
```

Comment:
There are already tests for this (+json) - what is different here is the explicit charset; i would tweak test name accordingly.

Reply:
Or merge tests?